model-optimizer/mo/utils/summarize_graph.py

```diff
@@ -70,9 +70,10 @@ def summarize_graph(graph_def):
     if argv.input_model and argv.saved_model_dir:
         print("[ ERROR ] Both keys were provided --input_model and --input_dir. Please, provide only one of them")
         sys.exit(1)
-    graph_def, _ = load_tf_graph_def(graph_file_name=argv.input_model, is_binary=not argv.text,
-                                     checkpoint=argv.input_checkpoint,
-                                     model_dir=argv.saved_model_dir, saved_model_tags=argv.saved_model_tags)
+    tags = argv.saved_model_tags.split(",")
```

Comment:
Please add check if argv.saved_model_tags not None

Reply:
This key has a default value, and this default value is an empty string (see lines 61-63 of the file `summarize_graph.py`).